api/routers/internal.py

```
@@ -244,6 +244,30 @@ async def list_hosts(username: str):
+    Consumed by the sshd-side account reconciler (user-sync.sh) to pre-create OS
+    accounts before a user's first SSH connection. OpenSSH will not invoke
+    AuthorizedKeysCommand for a username that does not already resolve to an OS
+    account, so the account must exist up front. Only real, key-bearing users
+    are returned, which bounds /etc/passwd growth to legitimate accounts.
+    """
+    factory = get_session_factory()
+    async with factory() as db:
+        result = await db.execute(
+            select(User.username)
+            .join(SSHKey, SSHKey.user_id == User.id)
+            .where(User.is_active == True, SSHKey.is_active == True)
```

Ruff (0.15.21)

[error] Avoid equality comparisons to `True`; use `User.is_active:` for truth checks

Replace with `User.is_active`

(E712)